modules/static/protocolbuffers/wellknowntypes/pre-sync.sh

```diff
@@ -0,0 +1 @@
+cp java/core/src/main/resources/google/protobuf/java_features.proto src/google/protobuf
```

Comment:
This is failing when I run the fetch script locally (edited fetch.sh to comment out syncing all other modules at the bottom and comment out the line about `BUF_TOKEN` at the top).

We need to do a `pushd "${module_root}/${git_owner}/${git_repo}" > /dev/null` / `popd > /dev/null` either in the main fetch script around the call to `pre-sync.sh` or in the `pre-sync.sh` script itself to run within the checked out git directory for the module.

Reply:
My bad. I thought it was already in the current directory since I saw the use of `.` with the rsync command. I think maybe I just need to move this to after any existing directory change. I'll play around with it locally first.

Thanks for taking time to poke at it.

Reply:
Okay, I did the same things you did to test it, and it appears to be working now. I've produced an alternate version of the 26.1 tag.

I'm going to _remove_ that change (the updated 26.1) from this PR so that we could land this and be ready for v27.0. And then I'll open a separate PR that modifies the `state.json` file for v26.1, and we can continue investigating if it's possible to apply that without breaking the "release" and "sync" steps of the process. In theory, since tags are now movable in the BSR, it should be possible to apply the change and move the tag to the updated commit. But the "release" and "sync" code may make assumptions about already-sync'ed versions of modules...